### ISSUE TITLE
Clarify which libraries are in which state

### ISSUE THREAD

**Issue by jducoeur** [typelevel/governance#55]
We have a concept of "primary" vs "satellite" libraries, largely based on whether the library is more managed by its originator vs TL as a whole.  We should make sure that:

* We have a crisp statement about what these concepts mean.
* We know exactly which libraries fall into which categories.

Note that probably the large majority of the ecosystem falls into the larger satellites.  (Eg, Davenverse and http4s are each north of 40 libraries.)  We should think about how to present these as a coherent ecosystem while still giving the individual projects their due, but step one is making sure we know what they are.

**Comment by rossabaker** [comment]
There are two levels described in the charter: Organization Projects (section 8) and Affiliate projects (section 9).  This replaces the old "member" vs. "incubator" delineation.  The newer model is not yet explained anywhere in non-charter speak.

https://github.com/typelevel/governance/blob/main/CHARTER.md#8-project-criteria

**Comment by reardonj** [comment]
I think this is now reasonably described on the site? https://typelevel.org/projects/

**Comment by jducoeur** [comment]
Yep, agreed -- I think the wording on that page satisfies the intent of this Issue.
